servers/services/src/main/java/org/projectnessie/services/cel/CELUtil.java

```diff
@@ -69,12 +78,122 @@
           Decls.newVar(VAR_ROLE, Decls.String),
           Decls.newVar(VAR_OP, Decls.String));
 
-  public static final List<Object> COMMIT_LOG_TYPES = Collections.singletonList(CommitMeta.class);
+  public static final List<Object> COMMIT_LOG_TYPES =
+      ImmutableList.of(CommitMeta.class, OperationForCel.class, ContentKey.class, Namespace.class);
 
   public static final List<Object> REFERENCES_TYPES =
       ImmutableList.of(CommitMeta.class, ReferenceMetadata.class, Reference.class);
 
   public static final CommitMeta EMPTY_COMMIT_META = CommitMeta.fromMessage("");
   public static final ReferenceMetadata EMPTY_REFERENCE_METADATA =
       ImmutableReferenceMetadata.builder().commitMetaOfHEAD(EMPTY_COMMIT_META).build();
+
+  /**
+   * 'Mirrored' interface wrapping a {@link Operation} for CEL to have convenience fields for CEL
+   * and to avoid missing fields due to {@code @JsonIgnore}.
+   */
+  @SuppressWarnings("unused")
+  public interface OperationForCel {
```

Comment:
Could this be a class instead? It looks like we only have one impl. and it would be nice to keep the runtime logic together with the getter method definitions, IMHO.